src/main/resources/application-custom.yaml

```diff
@@ -1,4 +1,19 @@
+#Adds the option to go to eg. http://localhost:8080/actuator/health for seeing the running configuration
+#see https://docs.spring.io/spring-boot/docs/current/reference/html/actuator.html#actuator.endpoints
+management:
+  endpoints:
+    web:
+      exposure:
+        include: "health,prometheus"
 spring:
+  main:
+    allow-circular-references: true
+    allow-bean-definition-overriding: true
```

Comment:
please double-check all of the changes to the `application.yaml` file. this line is commented-out in the upstream repo.